package.json

```diff
@@ -77,17 +76,15 @@
     "test": "jest",
     "clean": "ocular-clean all",
     "build": "ocular-clean all && ocular-build --dist esm && lerna run build",
-    "publish": "ocular-publish",
+    "publish": "npm run lerna:publish",
     "-------- LERNA --------": "",
     "bump": "npm run lerna:bump",
     "lerna": "lerna",
     "lerna:depcheck": "lerna exec --since master --parallel -- depcheck",
     "lerna:bump": "lerna version --no-push && npm run lerna:sync && npm run lerna:git",
     "lerna:sync": "jq -M --argfile lerna lerna.json '.version = $lerna.version' package.json > tmp.json && mv tmp.json package.json",
     "lerna:git": "git add package.json && git commit --amend",
-    "lerna:build": "lerna run build",
-    "lerna:clean": "lerna run clean",
-    "lerna:publish": "lerna run publish",
+    "lerna:publish": "lerna exec cp package.json dist && lerna exec npm publish dist",
```

Comment:
Used `lerna exec` to publish `/dist` folders only

Reply:
same as above, use `yarn publish` instead.